src/xbot/common/math/PIDManager.java

```diff
@@ -19,6 +19,7 @@
     private DoubleProperty maxOutput;
     private DoubleProperty minOutput;
     private BooleanProperty isEnabled;
+    private boolean isIMasked = false;
```

Comment:
It's funny because in normal English people would absolutely get the idea that "masking" is "hiding", but in CS it's a little less clear =]